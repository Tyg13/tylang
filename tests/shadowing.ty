fn printf(fmt: str, ...) -> i32;
fn foo(a: i32) -> i32 {
    let a: i16 = a as i16;
    let b: i16 = a + 1;
    let b: i32 = (b as i32) + 10;
    b
}

fn main(argc: i32, argv: *str) -> i32 {
    printf("foo(10) == %d\n", foo(10));
    0
}
