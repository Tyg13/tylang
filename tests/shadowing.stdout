foo(10) == 21
